lib/generator._js

```diff
@@ -95,6 +96,18 @@ function ScriptGenerator(repositoryRoot, projectType, projectPath, solutionPath,
     var relativeSolutionPath = path.relative(repositoryRoot, solutionPath);
     log.info('Solution file path: .' + path.sep + relativeSolutionPath);
     this.solutionPath = relativeSolutionPath;
+
+    // find .xproj file, should be under projectPath
+    var files = fs.readdirSync(projectPath);
+    var webProjFiles = files.filter(function(file){
+      return file.endsWith(".xproj");
```

Comment:
Thanks @mejje 
@watashiSHUN let's add a [polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith#Polyfill) for endsWith if you wanna use it.